test/lib.sh

```diff
@@ -59,3 +59,98 @@ EOF
 hr() {
     echo "========================="
 }
+
+## diff BASELINE
+#
+# Recursively compare the current repository content to the BASELINE repository
+#
+# :param BASELINE: An existing directory used as reference to compared sync'ed files
+diff() {
+	for file in `ls` ; do 
+		if [ -d $file ] ; then 
+			pushd $file  > /dev/null
+			diff $1/$file 
```

Comment:
because `diff --recursive` would not allow to know how many files differs and I believe this is meaningful information for debugging. After, we can say we don't want/need this information but I think it would be helpful (and eventually, add the list of different files in the check).

Reply:
checking around, this can be worked around with the `-q` option and having/generating a log file to have the list